toonz/sources/translations/spanish/toonz.ts

```diff
@@ -7063,7 +7059,7 @@ También establecer la ruta de salida a esa carpeta.</translation>
         <translation type="vanished">   Color de fotogramas anteriores: </translation>
     </message>
     <message>
-        <source>Display Lines Only </source>
+        <source>Display Lines Only</source>
```

Comment:
You do not correct `source` typos here. You need to correct them in code.  Text is made translatable in code with the `tr()` function.

There is a script that is run which updates the `source` data in these .ts files. Afterwards, a member of the community who speaks that language natively will correct the `translation` data. If you modify it here but not in code, the changes will get lost when we run the script which usually happens just before a release.

Reply:
Also, after the .ts files are updated, you need to generate the .qm files and place them in the stuff/config/loc language folder.